fuzz/parse.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+OUTPUT_DIR=$1
+
+screen -S grammar_fuzzer -d -m  /bin/bash -c "AFL_CUSTOM_MUTATOR_LIBRARY=/usr/local/lib/libgrammarmutator-ruby.so AFL_CUSTOM_MUTATOR_ONLY=1 afl-fuzz -i fuzz/corpus/parse -o $OUTPUT_DIR -S grammar -- ./build/fuzz.parse"
```

Comment:
Seems like the parse corpus is a direct copy of the tests from the top level. Could we do something to remove this duplication like mount that as a directory in docker instead?

Reply:
Yes, that'd work. Or a symlink. I was wondering if there's a better collection of programs I can use as a corpus? This was the laziest possible thing I could do...

Reply:
We should probably use our fixtures. They're the best corpus we have. Everything under `test/fixtures/**/*.txt` is a Ruby program. We have them listed as text only to not confuse tools looking for all Ruby programs.

Reply:
That's far more sensible! I'll try that out.